deploy-manage/manage-connectors.md

```diff
@@ -2,25 +2,98 @@
+
+Use the [action configuration settings](https://www.elastic.co/guide/en/kibana/current/alert-action-settings-kb.html#action-settings) to customize connector networking configurations, such as proxies, certificates, or TLS settings. You can set configurations that apply to all your connectors or use `xpack.actions.customHostSettings` to set per-host configurations.
+
+## Connector list [connectors-list]
+
+In **{{stack-manage-app}} > {{connectors-ui}}**, you can find a list of the connectors in the current space. You can use the search bar to find specific connectors by name and type. The **Type** dropdown also enables you to filter to a subset of connector types.
+
+:::{image} ../images/kibana-connector-filter-by-type.png
+:alt: Filtering the connector list by types of connectors
+:class: screenshot
+:::
+
+You can delete individual connectors using the trash icon. Alternatively, select multiple connectors and delete them in bulk using the **Delete** button.
+
+:::{image} ../images/kibana-connector-delete.png
+:alt: Deleting connectors individually or in bulk
+:class: screenshot
+:::
+
+::::{note}
+You can delete a connector even if there are still actions referencing it. When this happens the action will fail to run and errors appear in the {{kib}} logs.
+
+::::
+
+## Creating a new connector [creating-new-connector]
+
+New connectors can be created with the **Create connector** button, which guides you to select the type of connector and configure its properties. For a full list of available connectors, see [Available connectors](asciidocalypse://docs/kibana/docs/reference/connectors-kibana/connectors-kibana.md)
+
+::::{note}
+Some connector types are paid commercial features, while others are free. For a comparison of the Elastic subscription levels, go to [the subscription page](https://www.elastic.co/subscriptions).
+::::
+
+:::{image} ../images/kibana-connector-select-type.png
+:alt: Connector select type
+:class: screenshot
+:::
+
+After you create a connector, it is available for use any time you set up an action in the current space.
+
+For out-of-the-box and standardized connectors, refer to [preconfigured connectors](https://www.elastic.co/guide/en/kibana/current/pre-configured-connectors.html).
+
+::::{tip}
+You can also manage connectors as resources with the [Elasticstack provider](https://registry.terraform.io/providers/elastic/elasticstack/latest) for Terraform. For more details, refer to the [elasticstack_kibana_action_connector](https://registry.terraform.io/providers/elastic/elasticstack/latest/docs/resources/kibana_action_connector) resource.
+::::
+
+## Importing and exporting connectors [importing-and-exporting-connectors]
+
+To import and export connectors, use the [Saved Objects Management UI](/explore-analyze/find-and-organize/saved-objects.md).
+
+:::{image} ../images/kibana-connectors-import-banner.png
+:alt: Connectors import banner
+:class: screenshot
+:::
```

Comment:
this image is even more large

consider the value of the images: do we need them?